insights/parsers/vdo_status.py

```diff
@@ -0,0 +1,209 @@
+"""
+VdoStatus - command ``/usr/bin/vdo status``
+===========================================
+
+Module for parsing the output of command ``vdo status``. The bulk of the
+content is split on the colon and keys are kept as is.
+
+"""
+
+from __future__ import division
+from insights import parser
+from insights import YAMLParser
+from insights.specs import Specs
+from insights.parsers import ParseException
+
+
+@parser(Specs.vdo_status)
+class VDOStatus(YAMLParser):
```

Comment:
In the title, you have used `VdoStatus`, please uniform it in both the doc and code.